codice/ddf
# DDF-4292 Allowing Intrigue's Coordinate Display to be Configureable

#### What does this PR do?
Allows the Coordinate Display inside Intrigue on the bottom-left to be configurable inside Catalog UI Search under the Summary Metacard Attributes section. Thus the Coordinate Display can now show information that was previously contained inside the Inspector view.

![configuring](https://user-images.githubusercontent.com/23177404/48096460-70684c80-e1d4-11e8-8561-58655337a3f3.gif)

![coodinates](https://user-images.githubusercontent.com/23177404/48096733-39466b00-e1d5-11e8-99e0-4c1ef19bd32e.gif)


#### Who is reviewing it? 

@bdeining 
@Schachte 
@adimka 
@middlets719 
@gordocanchola 
@andrewkfiedler 
@djblue 

#### Select relevant component teams: 
@codice/ui 

#### Ask 2 committers to review/merge the PR and tag them here.

@andrewkfiedler
@bdeining
@djblue

#### How should this be tested?
Go to Catalog UI Search and add/remove attributes as desired.
Go to Intrigue and perform a search. 
Hover over any icons or geos and check to see that the desired attributes are being shown in the Coordinates Display.

#### Any background context you want to provide?
#### What are the relevant tickets?
[DDF-4292](https://codice.atlassian.net/browse/DDF-4292)

#### Screenshots

#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
